js_modules/dagster-ui/packages/ui-components/src/components/CursorControls.tsx

```diff
@@ -4,6 +4,7 @@ import {Button} from './Button';
 import {Icon} from './Icon';
 
 export interface CursorPaginationProps {
+  cursor?: string;
```

Comment:
Added this so that the parent component can determine when the cursor has changed, and reset scroll position accordingly.